packages/azure/src/index.mts

```diff
@@ -23,3 +23,19 @@ app.http('securityWatcher', {
     authLevel: 'anonymous',
     handler: securityWatcher
 });
+
+app.http('checkConfig', {
+  methods: ['GET'],
+  authLevel: 'anonymous',
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  handler: async function (_request: HttpRequest, _context: InvocationContext): Promise<HttpResponseInit> {
+
+    const requiredEnvVars = ["APP_ID", "PRIVATE_KEY", "GITHUB_CLIENT_ID" , "GITHUB_CLIENT_SECRET", "WEBHOOK_SECRET"];
```

Comment:
You are right, Client ID and secret are only needed for OAuth

Will change the PRIVATE_KEY later if it's needed for ACA, but using a mount instead of an environment variable might be unnecessary complexity. Will cross that bridge when we reach it.